calcom/cal.com
# refactor: Eventtype app context after orgs

## What does this PR do?



This is based on #9950.

This PR refactors the eventtypeAppContext from an array to an object to grab only what is needed.

Fixes # (issue)



## Requirement/Documentation



- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change




## How should this be tested?




- Enable event type apps
- Go to the event type app page
- Install apps
    - Ensure that changes can be made and saved
- Create a managed event type
    - Ensure that the owner can make changes to the event type apps
    - Go to the child event type and ensure that changes can not be made

## Mandatory Tasks


## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
